packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -168,6 +174,7 @@ function Cell(props) {
       className={className}
       onMouseDown={handleMouseDown}
       onKeyDown={handleKeyDown}
+      style={{ left: columnContext.pinnedColumnOffsets[columnIndex] }} // eslint-disable-line react/forbid-component-props
```

Comment:
We would only add this style if the cell is in a pinned column.  We should not rely on access elements beyond the bounds of the array for our implementation.

Reply:
[69036d9](https://github.com/cerner/terra-framework/pull/1710/commits/69036d994769ea38abc772f887a266894c8f9517)